mis_builder/models/mis_report.py

```diff
@@ -317,11 +323,12 @@ def name_get(self):
                 )
             else:
                 name = rec.kpi_id.display_name
-            res.append((rec.id, name))
+            rec.display_name = name
+            res.append(rec)
         return res
 
     @api.model
-    def name_search(self, name="", args=None, operator="ilike", limit=100):
+    def _name_search(self, name="", args=None, operator="ilike", limit=100):
```

Comment:
```suggestion
    def _name_search(self, name, domain=None, operator='ilike', limit=None, order=None):
```

This is the signature of `_name_search` in BaseModel.